packages/ui/src/theme/css/component/liveness.scss

```
@@ -59,6 +59,9 @@
   left: 0;
   width: 100%;
   height: 100%;
+}
+
+.amplify-liveness-video-user-facing {
```

prefer bem style modifier here: 
```suggestion
.amplify-liveness-video--user-facing {
```
https://getbem.com/naming/